### ISSUE TITLE
Jump to heading

### ISSUE THREAD

**Issue by gandalfsaxe** [typora/typora-issues#1591]
I'd be wonderful to have a "Jump to heading..." feature, similar to "Quick open", just for headings within the document. Should use fuzzy search. See [Caret](url) for an excellent implementation.

**Comment by gandalfsaxe** [comment]
Also similar to "Go to Symbol in File..." in VSCode and equivalent in many other editors.

This is literally still the feature I miss the most :)

**Comment by AlexPasternak** [comment]
I agree that doesn't sound very useful. However, @gandalfsaxe, your Caret link is broken in the original post, so may make it harder to see what you're requesting.

**Comment by gandalfsaxe** [comment]
@AlexPasternak Does or doesn't sound useful? 😄 

I fixed the link and added a GIF showing it in use 🙂

**Comment by DutchPete** [comment]
Isn't this issue the same as a link to a header like so
[link text](#header-text)

**Comment by GandalfSaxe** [comment]
@DutchPete What do you mean? Linking to headers works fine in Typora.

**Comment by DutchPete** [comment]
@GandalfSaxe: yes, linking to headers works fine, I just discovered.
But what is the difference with "Jump to heading..." that you have requested?

**Comment by GandalfSaxe** [comment]
I'm suggesting a searchbox where you can fuzzy search all the headers and jump to them instantly, basically instead of scrolling or "Find..." which will search ALL content, not just headers.

Linking to headers is entirely different because it's about linking content to navigate to a header. I'm talking about navigation only. See the GIF in the first post for an example of how it works.

**Comment by DutchPete** [comment]
OK GandalfSaxe, it's clear what you mean. Thank you.

**Comment by Ciantic** [comment]
Quick Outline

I just wanted to add a search keyword here 👍
